partner_invoicing_mode/models/res_company.py

```diff
@@ -10,5 +10,4 @@ class ResCompany(models.Model):
     invoicing_mode_standard_last_execution = fields.Datetime(
         string="Last execution (standard)",
         help="Last execution of standard invoicing.",
-        readonly=True,
```

Comment:
Not sure I understand this change.
I can't think of a reason why we'd want tthis field to be imported